l3kernel/l3luatex.dtx

```diff
@@ -358,13 +359,16 @@ local package_loaded    = package.loaded
 local package_searchers = package.searchers
 local table_concat      = table.concat
 
-local scan_int     = token.scan_int or token.scan_integer
-local scan_string  = token.scan_string
-local scan_keyword = token.scan_keyword
-local put_next     = token.put_next
-local token_create = token.create
-local token_new    = token.new
-local set_macro    = token.set_macro
+local scan_csname   = token.scancsname or token.scan_csname
+local scan_int      = token.scan_int or token.scan_integer
+local scan_string   = token.scan_string
+local scan_keyword  = token.scan_keyword
+local put_next      = token.put_next
+local token_create  = token.create
+local get_macro     = token.getmacro or token.get_macro
+local get_protected = token.getprotected or token.get_protected
+local token_new     = token.new
+local set_macro     = token.set_macro
```

Comment:
Might as well add the other non-underscored names while you're editing this part of the file.

```suggestion
local scan_string   = token.scanstring or token.scan_string
local scan_keyword  = token.scankeyword or token.scan_keyword
local put_next      = token.putnext or token.put_next
local token_create  = token.create
local get_macro     = token.getmacro or token.get_macro
local get_protected = token.getprotected or token.get_protected
local token_new     = token.new
local set_macro     = token.setmacro or token.set_macro
```